<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configure</class>
 <widget class="QDialog" name="Configure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Audio">
    <attribute name="title">
     <string>Audio</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Display">
    <attribute name="title">
     <string>Display</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Spectrum</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>High:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Low:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spectrumHighSpinBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-200</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spectrumLowSpinBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-200</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>-160</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>231</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Waterfall</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>High:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Low:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="waterfallHighSpinBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-200</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>-80</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="waterfallLowSpinBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-200</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>-120</number>
      </property>
     </widget>
    </widget>
    <widget class="QSpinBox" name="fpsSpinBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>50</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>40</number>
     </property>
     <property name="value">
      <number>15</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>FPS:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Configure</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Configure</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
